TechDraw/LasercutterSVGExport.FCMacro

```diff
@@ -205,117 +39,20 @@ class LasercutterTechdrawExport():
             return(True)
         else:
             return(False)
-        
-    def GetResources(self):
-        '''Return the icon which will FreeCADear in the tree view. This method is optional and if not defined a default icon is shown.'''
-        return {'Pixmap'  : os.path.join(iconPath, 'LasercutterTechdrawExport.svg'),
-                'Accel' : '', # a default shortcut (optional)
-                'MenuText': 'Lasercutter Techdraw Export',
-                'ToolTip' : __doc__ }
                 
     def makeExport(self):          
         # Generate commands in the FreeCAD python console to create LasercutterTechdrawExport
-        gui.doCommand("from LasercutterTechdrawExport import LasercutterTechdrawExport")
-        
+        gui.doCommand("from LasercutterSVGExport import LasercutterTechdrawExport")
+
         gui.doCommand("lcparts = []")
         selection = gui.Selection.getSelectionEx()
         for sel in selection:
             gui.doCommand("lcparts.append(FreeCAD.ActiveDocument.getObject('%s'))"%(sel.ObjectName))
             
         gui.doCommand("LasercutterTechdrawExport.makeLasercutterTechdrawExport(lcparts, BeamWidth = %f, doc = App.activeDocument())"%self.form.doubleSpinBox.value())
```

Comment:
Do you know `gui.doCommand(f'LasercutterTechdrawExport.makeLasercutterTechdrawExport(lcparts, BeamWidth={self.form.doubleSpinBox.value()}, doc=FreeCAD.activeDocument())')`?

I mainly wanted to mention formatted strings but I changed a few other details according to Python standard and consistency between the use of `FreeCAD` and `App`.